bibliopixel/remote/control.py

```diff
@@ -57,10 +58,17 @@ def __init__(self, config, animations):
             anim_cfg = dict(DEFAULT_ANIM_CONFIG)
             anim_cfg.update(anim)
             if not anim_cfg['display']:
-                anim_cfg['display'] = anim['name']
+                anim_cfg['display'] = anim_cfg['name']
             anim_cfg['name'] = ''.join(e for e in anim['name'] if e.isalnum())
-            anim_cfg['animation'].on_completion = self.on_completion
-            self.animation_objs[anim_cfg['name']] = anim_cfg['animation']
+
+            if anim_cfg['animation'] is None:
```

Comment:
I can't delete it first... I have to check if it showed up as None (which means it didn't load) and then I can get rid of it. Your way means it never loads :P